samples/samples/pg_snippets.py

```diff
@@ -1681,6 +1690,9 @@ def insert_customers(transaction):
             )
         )
 
+    instance = spanner_client.instance(instance_id)
```

Comment:
why do we need this handwritten objects?

Reply:
We need this to do data client inserts.